qtazu/widgets/login.py

```diff
@@ -110,15 +178,24 @@ def on_login(self):
 
         try:
             gazu.set_host(host)
+            if not gazu.client.host_is_up():
+                raise ConnectionError(
+                    "Could not connect to the server. Is the host URL correct ?"
```

Comment:
Let's remove the space between the sentence and the question mark, like `Is the host URL correct?`.
Same for the message below.